framework/PageForm/GenericForm.tsx

```diff
@@ -0,0 +1,70 @@
+import { useContext, ReactNode, BaseSyntheticEvent } from 'react';
```

Comment:
Do we really need GenericForm? I'd like to keep it so that we do not have 2 ways to do the same thing.

Reply:
It gets us the same function form behavior as `PageForm`, but without the layout behaviors -- which is exactly what the login form needs.